packages/e2e-tests/specs/editor/plugins/plugins-api.test.js

```diff
@@ -156,4 +159,16 @@ describe( 'Using Plugins API', () => {
 			expect( pluginDocumentSettingsText ).toMatchSnapshot();
 		} );
 	} );
+
+	describe( 'Block Settings Menu Item', () => {
+		it( 'Should render a new item', async () => {
+			await insertBlock( 'List' );
+			await page.keyboard.type( 'one' );
+			await page.keyboard.press( 'Enter' );
+			await clickBlockToolbarButton( 'Options' );
+			await clickMenuItem( 'My new plugin' );
+
+			expect( console ).toHaveLoggedWith( 'Block clicked' );
```

Comment:
According to the report https://github.com/WordPress/gutenberg/pull/20357/checks?check_run_id=3251810969 it looks like the console.log was not called. I can see this working fine in interactive mode, though.

Reply:
Updated it to `toHaveLogged` locally and it also fails, so I wonder if I have to do anything special to test console messages.